packages/frontend/src/components/ControlledAutocomplete/index.tsx

```diff
@@ -29,7 +29,10 @@ const formComboboxOptions = (
   const result: ComboboxItem[] = []
   for (const option of options) {
     const item = {
-      value: option['value'],
+      value:
+        typeof option['value'] === 'number'
+          ? JSON.stringify(option['value']) // Only stringify numbers
```

Comment:
Can we discuss this more in office tomorrow

Reply:
ok lemme dm you on this issue

Reply:
Ok problem is quite complex due to `freeSolo`. Let's stick with this. Can we update the comment to explain why we have this use case. Secondly we also need to make all `freeSolo` inputs typed as strings.